vmdb/app/models/host.rb

```diff
@@ -1751,6 +1753,7 @@ def scan_from_queue(taskid=nil)
     $log.info("#{log_header} Scanning #{log_target}...")
 
     task.update_status("Active", "Ok", "Scanning") if task
+    self.authentication_status_by_parent = "Valid" if self.kind_of?(HostOpenstackInfra) && self.verify_credentials
```

Comment:
I couldn't figure out why this was tripping me up before.  The problem I'm seeing is that `authentication_status_by_parent` is never set to anything other than `"Valid"` or `nil`.  If the `authentication_status_by_parent` is ever set to `"Valid"`, it is forever valid, even if the parent's authentication is invalidated.